lib/redis/connection/ruby.rb

```diff
@@ -237,6 +237,14 @@ def self.connect(path, timeout)
       class SSLSocket < ::OpenSSL::SSL::SSLSocket
         include SocketMixin
 
+        def wait_readable(timeout = nil)
```

Comment:
As mentioned, if you plan to add this to SSLSocket, should we put this under a `unless method_define?(:wait_readable)`?

Reply:
I am okay with that, but frankly it will never matter, this will be identical the implementation we put into OpenSSL.

Reply:
I've added this in 0a9ae6b.